src/web/components/Onwards/Onwards.test.tsx

```diff
@@ -19,7 +19,8 @@ describe('OnwardsLayout', () => {
         useApi.mockReturnValue(response);
 
         const { asFragment } = render(
-            <OnwardsLayout onwardSections={[linkAndDescription]} />,
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            <OnwardsLayout {...linkAndDescription} />,
```

Comment:
more of a preference, but I like to avoid using spread operator when possible (same for the stories)

Reply:
Yeah, I generally agree. I wanted `Carousel` and `OnwardsLayout` to have the same interface so that the calling code could treat them generically. The simplest interface was just to accept an `OnwardsType`, which has a lot of fields. Is the motivation here that avoiding spread is more explicit about dependencies?

Reply:
yeah, its more that I want to discourage the use of spread operator in general for react components, as its can make it a little more challenging to follow the props when debugging. Also future developers may look at `test/storybook` code to see how to implement things; and this could invite them to use this approaches. Thats why I would personally not have used spread. But again this is just preference.